server/app.ts

```diff
@@ -34,10 +42,21 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpWebRequestParsing())
   app.use(setUpStaticResources())
   nunjucksSetup(app, path)
-  app.use(setUpAuthentication())
-  app.use(authorisationMiddleware())
+
+  app.use(
+    setUpAuthentication(),
+    rateLimiter({ windowMs: authenticationRateLimitWindow, maxRequests: authenticationRateLimitMaxRequests }),
+  )
+  app.use(
+    authorisationMiddleware(),
+    rateLimiter({ windowMs: authorisationRateLimitWindow, maxRequests: authorisationRateLimitMaxRequests }),
+  )
+  app.use(
+    setUpCurrentUser(services),
+    rateLimiter({ windowMs: authorisationRateLimitWindow, maxRequests: authorisationRateLimitMaxRequests }),t
```

Comment:
typo here: `t`

Reply:
Whoops! I had some Prettier weirdness going on in my IDE, and rather than diagnose it, I made the changes in TextEdit 🤦